### PR TITLE
:globe_with_meridians: N°7687 Update German translations for iTop 3.2

### PR DESCRIPTION
**Pull request by larhip**
Updated German translations thanks to @Attila0428

### CODE REVIEW

File: dictionaries/de.dictionary.itop.core.php [hunk 1]
```diff
@@ -111,39 +125,32 @@
...
-	'Class:ActionNewsroom/Attribute:title' => 'Title~~',
-	'Class:ActionNewsroom/Attribute:title+' => 'Title of the news. Can contain placeholders like $this->attribute_code$~~',
-	'Class:ActionNewsroom/Attribute:url' => 'URL~~',
-	'Class:ActionNewsroom/Attribute:url+' => 'If empty, the URL will point to the object triggering the notification. But you can also specify a custom URL.~~',
+	'Class:ActionNewsroom' => 'Benachrichtigung durch Newsroom',
+	'Class:ActionNewsroom+' => '',
+	'Class:ActionNewsroom/Attribute:icon' => 'Icon',
+	'Class:ActionNewsroom/Attribute:icon+' => '',
+	'Class:ActionNewsroom/Attribute:language' => 'Sprache',
+	'Class:ActionNewsroom/Attribute:language+' => '',
```
Comment (Molkobain) [inline dictionaries/de.dictionary.itop.core.php]: Attribute descriptions have been removed, could you please either revert them in english or translate them?

File: dictionaries/de.dictionary.itop.core.php [hunk 2]
```diff
@@ -111,39 +125,32 @@
...
+	'Class:ActionNewsroom/Attribute:priority/Value:1' => 'Kritisch',
+	'Class:ActionNewsroom/Attribute:priority/Value:1+' => 'Kritisch',
+	'Class:ActionNewsroom/Attribute:priority/Value:2' => 'Dringend',
+	'Class:ActionNewsroom/Attribute:priority/Value:2+' => 'Dringend',
+	'Class:ActionNewsroom/Attribute:priority/Value:3' => 'Wichtig',
+	'Class:ActionNewsroom/Attribute:priority/Value:3+' => 'Wichtig',
+	'Class:ActionNewsroom/Attribute:priority/Value:4' => 'Standard',
+	'Class:ActionNewsroom/Attribute:priority/Value:4+' => 'Standard',
+	'Class:ActionNewsroom/Attribute:recipients' => 'Empfänger',
+	'Class:ActionNewsroom/Attribute:recipients+' => '',
```
Comment (Molkobain) [inline dictionaries/de.dictionary.itop.core.php]: Same comment

File: dictionaries/de.dictionary.itop.core.php [hunk 3]
```diff
@@ -111,39 +125,32 @@
...
+	'Class:ActionNewsroom/Attribute:test_recipient_id' => 'Testempfänger',
+	'Class:ActionNewsroom/Attribute:test_recipient_id+' => 'Person für Testbenachrichtigungen (statt regulärer Empfänger)',
+	'Class:ActionNewsroom/Attribute:title' => 'Titel',
+	'Class:ActionNewsroom/Attribute:title+' => '',
```
Comment (Molkobain) [inline dictionaries/de.dictionary.itop.core.php]: Same comment

File: dictionaries/de.dictionary.itop.core.php [hunk 4]
```diff
@@ -111,39 +125,32 @@
...
+	'Class:ActionNewsroom/Attribute:url' => 'URL',
+	'Class:ActionNewsroom/Attribute:url+' => '',
```
Comment (Molkobain) [inline dictionaries/de.dictionary.itop.core.php]: Same comment

File: dictionaries/de.dictionary.itop.ui.php [hunk 5]
```diff
@@ -1463,8 +1463,4 @@
 	'UI_CountOfObjectsShort' => '%1$d Objekte.',
 	'portal:backoffice' => ITOP_APPLICATION_SHORT.' Backend',
 	'portal:legacy_portal' => 'Endbenutzer-Portal',
-	'Menu:ConfigEditor'           => 'General configuration~~',
-	'Menu:ConfigEditor+'          => 'Configuration File editor~~',
-	'Menu:Integrations' => 'Integrationen',
-	'Menu:Integrations+'          => '~~',
```
Comment (Molkobain) [inline dictionaries/de.dictionary.itop.ui.php]: Please restore these entries

File: dictionaries/ui/application/welcome-popup/de.dictionary.itop.welcome-popup.php [hunk 6]
```diff
@@ -1,54 +1,24 @@
...
-<div>Newsroom allows you to easily manage notifications within the platform, so you can stay on top of important updates without constantly checking your email. With the ability to mark messages as read or unread, and automatically delete old notifications, you have complete control over your notifications. </div>
-<br>
-<div>Try it out today and streamline your ' . ITOP_APPLICATION_SHORT . '\'s communication experience!</div>~~',
-	'UI:WelcomePopup:Message:320_03_NotificationsCenter:Title' => 'Notifications center~~',
-	'UI:WelcomePopup:Message:320_03_NotificationsCenter:Description' => '<div>As we know your information intake is already at its max, you can now easily choose how you receive your notifications - via email, chat, or even the Newsroom feature</div>
-<div>You don\'t want to receive a certain type of alerts? Nothing easier with these advanced customization capabilities giving you the flexibility to tailor your experience to your needs. </div>
+	'UI:WelcomePopup:Button:RemindLater' => 'Später erinnern',
+	'UI:WelcomePopup:Message:320_01:Description' => '<div>Willkommen bei '.ITOP_APPLICATION.' '.ITOP_VERSION_NAME.'!</div>
```
Comment (Molkobain) [inline dictionaries/ui/application/welcome-popup/de.dictionary.itop.welcome-popup.php]: Seems like you deleted the different messages. Please either revert them to english or translate them.

**Review by Molkobain — CHANGES_REQUESTED** [review]
Hello @larhip @Attila0428

Some description and welcome messages have been removed, could you please either restore them in english or translate them?

As a friendly reminder, the code freeze for 3.2.0 should be tonight.

**Comment by larhip** [comment]
HI @Molkobain thanks for your review!

when I remember correctly Attila did the translations based on the iTop professional beta build.
Unfortunatley I did not checked in detail and just created the PR, but it seems that in the meanwhile you changed a little bit in the core (added translations etc.) which leads to some missing or deleted translations..

I will check that hopefully that afternoon and update this PR

File: dictionaries/de.dictionary.itop.core.php (hunk 1 above)
Comment (larhip) [reply dictionaries/de.dictionary.itop.core.php]: ✅

File: dictionaries/de.dictionary.itop.core.php (hunk 2 above)
Comment (larhip) [reply dictionaries/de.dictionary.itop.core.php]: ✅

File: dictionaries/de.dictionary.itop.core.php (hunk 4 above)
Comment (larhip) [reply dictionaries/de.dictionary.itop.core.php]: ✅

File: dictionaries/de.dictionary.itop.ui.php (hunk 5 above)
Comment (larhip) [reply dictionaries/de.dictionary.itop.ui.php]: ✅

File: dictionaries/ui/application/welcome-popup/de.dictionary.itop.welcome-popup.php (hunk 6 above)
Comment (larhip) [reply dictionaries/ui/application/welcome-popup/de.dictionary.itop.welcome-popup.php]: ✅

**Comment by larhip** [comment]
@Molkobain 

Should be all done, might please check again?

Shall I rebase to latest develop or would you do so?

**Comment by Molkobain** [comment]
@larhip it's better thanks! But actually, is your branch based on develop or support/3.2 ?
It should be on support/3.2, otherwise the PR can only be merged in develop which is iTop 3.3.

If you need help to rebase it, feel free to ask. You must do a "rebase onto".

**Comment by Molkobain** [comment]
@larhip I can do it on my end, but as we can interfer with your fork it will the following consequences:
- PR will be marked as Closed as it cannot be merged as is
- Commit will still be authored by you, not me


What do you prefer?

**Comment by larhip** [comment]
I did already :-)

Rebase to support/3.2 should be done now, could you please check one more last time? :)

**Review by Molkobain — APPROVED** [review]

**Comment by Molkobain** [comment]
Perfect! Danke schöne Lars!

**Comment by larhip** [comment]
avec plaisir :-)

**Pull request merged**